### PR TITLE
Icelandic support

### PR DESCRIPTION
**Pull request by kumar-shridhar**

### CODE REVIEW

**Comment by kumar-shridhar** [comment]
Adding entities support for Icelandic language

**Comment by kumar-shridhar** [comment]
Any ideas why this test is failing?

My test is failing and in the CI Pipeline, I see this error:
IS Tests
Corpus Tests: FAIL (0.32s)
tests/Duckling/Testing/Asserts.hs:84:
empty result on "n\250ll"

I checked the "Assert.hs" file under IS and everything looks alright. I donot know why this is failing. Any clues, anyone ?

**Comment by patapizza** [comment]
Hey @kumar-shridhar, I'm on it, we'll get this merged soon. Hang tight!

**Comment by patapizza** [comment]
Þakka þér fyrir @kumar-shridhar!

File: Duckling/Rules.hs [hunk 1]
```diff
@@ -143,7 +142,6 @@ localeRules HI = HIRules.localeRules
 localeRules HR = HRRules.localeRules
 localeRules HU = HURules.localeRules
 localeRules ID = IDRules.localeRules
-localeRules IS = ISRules.localeRules
```
Comment (a2tm7a) [inline Duckling/Rules.hs]: @kumar-shridhar  Why did you remove these?
